### DISCUSSION TITLE
"You do not have permissions to do this" error

### DISCUSSION THREAD

**Discussion by Spleens1** [EvanMulawski/BeamMP.EnvSync#13] [Q&A] [answered]
I've set the mod up following the instruction but every time I send a message, I receive a direct message by my server: 
![image](https://user-images.githubusercontent.com/61100599/171051650-d17ae06e-c684-4ba2-aaf1-c52f3fb3db70.png)

I've added my User ID as admin but it still doesn't work.

**Comment by EvanMulawski** [comment, marked as answer]
Fixed in [v1.0.0-beta.3](https://github.com/EvanMulawski/BeamMP-EnvSync/releases/tag/v1.0.0-beta.3)

Thanks for the report.

**Reply by Spleens1** [reply]
Awesome! Thanks for the fast fix!
